lib/stripe/error.ex

```diff
@@ -166,67 +150,35 @@ defmodule Stripe.Error do
     end
   end
 
-  defp code_from_status(status) do
-    case status do
-      400 -> :bad_request
-      401 -> :unauthorized
-      402 -> :request_failed
-      404 -> :not_found
-      409 -> :conflict
-      429 -> :too_many_requests
-      s when s in [500, 502, 503, 504] -> :server_error
-      _ -> :unknown_error
-    end
-  end
-
-  defp generate_message(status, nil) do
-    case status do
-      400 -> "The request was unacceptable, often due to missing a required parameter."
-      401 -> "No valid API key provided."
-      402 -> "The parameters were valid but the request failed."
-      404 -> "The requested resource doesn't exist."
-      409 -> "The request conflicts with another request (perhaps due to using the same idempotent key)."
-      429 -> "Too many requests hit the API too quickly. We recommend an exponential backoff of your requests."
-      s when s in [500, 502, 503, 504] -> "Something went wrong on Stripe's end."
-      s -> "An unknown HTTP code of #{s} was received."
-    end
-  end
-
-  defp generate_message(_, type) do
-    case type do
-      :api_connection_error -> "The connection to Stripe's API failed."
-      :api_error -> "An internal Stripe error occurred. This is usually temporary."
-      :authentication_error -> "You failed to properly authenticate yourself in the request."
-      :card_error -> "The card could not be charged for some reason."
-      :invalid_request -> "Your request had invalid parameters."
-      :rate_limit_error -> "Too many requests hit the API too quickly. We recommend an exponential backoff of your requests."
-      :validation_error -> "A client-side library failed to validate a field."
-    end
-  end
-
-  # receives a map and a list of keys
-  # returns a map with all the provided keys, with either the original value in the input map
-  # or `nil` if the value was missing
-  defp maybe_pluck(map, keys) do
-    Enum.reduce(keys, %{}, fn key, acc ->
-      case map[key] do
-        nil -> acc
-        value -> Map.put(acc, key, value)
-      end
-    end)
-  end
```

Comment:
This exists because we tried to get an output of multiple variables in a very specific format. It also has a bug where the function does not actually behaves as described and simply performs the same thing `Map.take/1` would.

I removed the function and switched to individual `Map.get` calls instead, since the keys being read here are not used together, but separately.